bin/mustache_cli.ml

```diff
@@ -3,10 +3,15 @@ module Mustache = struct
   include With_locations
 end
 
-let apply_mustache json_data template_data =
+let apply_mustache ~json_data ~template_filename ~template_data =
   let env = Ezjsonm.from_string json_data
   and tmpl =
-    try Mustache.of_string template_data
+    let lexbuf = Lexing.from_string template_data in
+    let () =
+      let open Lexing in
+      lexbuf.lex_curr_p <- { lexbuf.lex_curr_p with pos_fname = template_filename };
```

Comment:
Note: this can now be done with just `Lexing.set_filename lexbuf template_filename`, but this function only appeared in 4.11 so I just went with the old way.